packages/hub/src/types/public.d.ts

```diff
@@ -79,3 +79,73 @@ export interface SpaceResourceConfig {
 	throttled?: boolean;
 	is_custom?: boolean;
 }
+
+export type License =
```

Comment:
will get stale over time but i guess it's no big deal?

Reply:
Ideally I'd like to move it to `@huggingface/tasks` eventually

Reply:
i'm not personally in favor of moving more stuff to `@huggingface` tbh – at least with current tooling (it's very painful to develop against)